<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slowwavedeviceClass</class>
 <widget class="QMainWindow" name="slowwavedeviceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>slowwavedevice</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/icon.png</normaloff>Resources/icon.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="OpenAction"/>
    <addaction name="SaveAction"/>
    <addaction name="SaveAsAction"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="ExitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="AboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="AbortAction"/>
   <addaction name="separator"/>
   <addaction name="OpenAction"/>
   <addaction name="SaveAction"/>
   <addaction name="SaveAsAction"/>
   <addaction name="ExitAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="slowwavedevice.qrc">
     <normaloff>:/slowwavedevice/Resources/start_icon_24x21.bmp</normaloff>:/slowwavedevice/Resources/start_icon_24x21.bmp</iconset>
   </property>
   <property name="text">
    <string>Запуск</string>
   </property>
   <property name="toolTip">
    <string>Запуск расчёта</string>
   </property>
  </action>
  <action name="OpenAction">
   <property name="icon">
    <iconset theme="document-open" resource="slowwavedevice.qrc">
     <normaloff>:/slowwavedevice/Resources/148705-essential-collection/png/folder-10.png</normaloff>:/slowwavedevice/Resources/148705-essential-collection/png/folder-10.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="icon">
    <iconset theme="document-save" resource="slowwavedevice.qrc">
     <normaloff>:/slowwavedevice/Resources/148705-essential-collection/png/save.png</normaloff>:/slowwavedevice/Resources/148705-essential-collection/png/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="SaveAsAction">
   <property name="icon">
    <iconset theme="document-save-as" resource="slowwavedevice.qrc">
     <normaloff>:/slowwavedevice/Resources/saveas.png</normaloff>:/slowwavedevice/Resources/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="icon">
    <iconset theme="application-exit" resource="slowwavedevice.qrc">
     <normaloff>:/slowwavedevice/Resources/148705-essential-collection/png/exit-1.png</normaloff>:/slowwavedevice/Resources/148705-essential-collection/png/exit-1.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="AboutAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="AbortAction">
   <property name="icon">
    <iconset resource="slowwavedevice.qrc">
     <normaloff>:/slowwavedevice/Resources/148705-essential-collection/png/stop.png</normaloff>:/slowwavedevice/Resources/148705-essential-collection/png/stop.png</iconset>
   </property>
   <property name="text">
    <string>Прервать</string>
   </property>
   <property name="toolTip">
    <string>Прервать вычисления</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="slowwavedevice.qrc"/>
 </resources>
 <connections/>
</ui>
